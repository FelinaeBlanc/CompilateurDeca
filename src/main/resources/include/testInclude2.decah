#include "testInclude.decah"

class B extends A{

    void display_B(){
        display_A(); 
        println("B OK"); 
    }

    float getX_B(){
        return (getX_A() + 1); 
    }
}