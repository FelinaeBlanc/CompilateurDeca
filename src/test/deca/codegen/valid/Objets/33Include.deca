#include "testInclude2.decah"

{
    B b = new B(); 
    b.display_A(); 
    b.display_B();
    printlnx(b.getX_A());
    printlnx(b.getX_B());
}