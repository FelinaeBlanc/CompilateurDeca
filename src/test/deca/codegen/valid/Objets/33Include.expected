0x1.681fa6p+5
0x1.681fa6p+5
B OK
0x1.681fa6p+5
0x1.701fa6p+5
